api/db/changelog/db.changelog-118-add-billing-account-to-workspaces.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="ericsong" id="db.changelog-118-add-billing-account-to-workspaces">
+    <addColumn tableName="workspace">
+      <column name="billing_account_name" type="varchar(100)">
+        <constraints nullable="false"/>
+      </column>
+    </addColumn>
+
+    <sql>update workspace set billing_account_name=(SELECT JSON_UNQUOTE(JSON_EXTRACT(configuration, '$.billing.accountId')) from config where config_id='main');</sql>
```

Comment:
If you want to be a bit more conservative (e.g. if you think perhaps some config objects might not have what you need), then you can use a temp table to hold {workspaceId, billing_acct} pairs and verify that looks good before committing a new column to the main table. I.e. if we get stuck, it's only this temporary (non-application) table that is in bad shape.